### PR TITLE
VZ-6867 Spec Changes from v1alpha1 to v1beta1 

### PR DESCRIPTION
**Pull request by absBitm**
Spec.EnvironmentName should be changed to Spec.Components.DNS.SubDomain
Spec.Components.Ingress should be changed to Spec.Components.IngressNGINX

### CODE REVIEW

File: platform-operator/apis/verrazzano/v1beta1/verrazzano_types.go [hunk 1]
```diff
@@ -314,9 +314,9 @@ type ComponentSpec struct {
 	// +optional
 	Grafana *GrafanaComponent `json:"grafana,omitempty"`
 
-	// Ingress contains the ingress-nginx component configuration
+	// IngressNGINX contains the ingress-nginx component configuration
 	// +optional
-	Ingress *IngressNginxComponent `json:"ingress,omitempty"`
+	IngressNGINX *IngressNginxComponent `json:"IngressNGINX,omitempty"`
```
Comment (anders-swanson) [inline platform-operator/apis/verrazzano/v1beta1/verrazzano_types.go:319]: JSON representation should be `ingressNGINX`

File: platform-operator/apis/verrazzano/v1beta1/verrazzano_types.go [hunk 2]
```diff
@@ -545,6 +545,7 @@ type DNSComponent struct {
 	// +optional
 	External         *External `json:"external,omitempty"`
 	InstallOverrides `json:",inline"`
+	SubDomain string `json:"subDomain"`
```
Comment (anders-swanson) [inline platform-operator/apis/verrazzano/v1beta1/verrazzano_types.go:548]: JSON representation must `omitempty`

**Review by anders-swanson — COMMENTED** [review]
Please make the JSON changes, regenerate the CRD/clients and commit.

**Review by anders-swanson — APPROVED** [review]

File: platform-operator/apis/verrazzano/v1alpha1/testdata/toallcomps/v1beta1.yaml [hunk 3]
```diff
@@ -120,7 +121,7 @@ spec:
             frobber: frob
     grafana:
       enabled: true
-    ingress:
+    ingressNGINX:
```
Comment (michael-cico) [inline platform-operator/apis/verrazzano/v1alpha1/testdata/toallcomps/v1beta1.yaml:124]: I feel like the JSON/YAML name should be `ingress-nginx` or `ingressNginx`.  Probably leaning towards the first one?

Comment (anders-swanson) [reply platform-operator/apis/verrazzano/v1alpha1/testdata/toallcomps/v1beta1.yaml:124]: `ingress-nginx` is best.

**Review by michael-cico — APPROVED** [review]

**Comment by anders-swanson** [comment]
@absBitm please pull master and also make the change in the v1beta1 profiles. EnvironmentName and Ingress must be updated there as well: https://github.com/verrazzano/verrazzano/tree/master/platform-operator/manifests/profiles/v1beta1

**Comment by anders-swanson** [comment]
Some UTs are also failing. Please fix Unit Tests.

**Pull request closed without merging**